### PR TITLE
feature: Trace query and debug enhancement

### PR DESCRIPTION
**Pull request by innerpeacez**
#### What type of this PR

Add one of the following kinds:
/kind feature

#### What this PR does / why we need it:
new trace feature

#### Specified Reviewers:

/assign @liuhaoyang @recallsong 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  Trace query and debug enhancement          |
| 🇨🇳 中文    |     链路追踪,链路调试功能优化   |

### CODE REVIEW

File: conf/msp/i18n/msp-i18n.yaml [hunk 1]
```diff
@@ -21,4 +34,17 @@ zh:
   workspace_prod: "生产"
   workspace_default: "默认"
   project_type_dop: "DevOps 项目"
-  project_type_msp: "微服务治理项目"
\ No newline at end of file
+  project_type_msp: "微服务治理项目"
+  span_count_desc: Span数量倒序
+  span_count_asc: Span数量正序
```
Comment (liuhaoyang) [inline conf/msp/i18n/msp-i18n.yaml:39]: Keep spaces between Chinese characters and letters

**Review by liuhaoyang — CHANGES_REQUESTED** [review]

File: modules/msp/apm/trace/core/query/query.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+type SpanTree map[string]*pb.Span
+type ConditionType string
+
+const (
+	INPUT ConditionType = "input"
+)
+
+var sortConditions = []*pb.TraceQueryCondition{
+	{Key: strings.ToLower(pb.SortCondition_TRACE_TIME_DESC.String()), Value: strings.ToLower(pb.SortCondition_TRACE_TIME_DESC.String())},
```
Comment (liuhaoyang) [inline modules/msp/apm/trace/core/query/query.go:33]: Use config.yml instead of hard coding

**Review by liuhaoyang — CHANGES_REQUESTED** [review]

File: modules/msp/apm/trace/core/query/query.go (hunk 2 above)
Comment (innerpeacez) [reply modules/msp/apm/trace/core/query/query.go:33]: After adding the global configuration reader, optimize the code

**Review by recallsong — APPROVED** [review]

**Review by liuhaoyang — APPROVED** [review]

**Pull request merged**